sycl/source/backend.cpp

```diff
@@ -95,6 +95,32 @@ queue make_queue_impl(pi_native_handle NativeHandle, const context &Context,
       std::make_shared<queue_impl>(PiQueue, ContextImpl, Handler));
 }
 
+queue make_queue_impl2(pi_native_handle NativeHandle, bool IsImmCmdList,
+                       const context &Context, RT::PiDevice Device,
+                       bool KeepOwnership, const property_list &PropList,
+                       const async_handler &Handler, backend Backend) {
+  const auto &Plugin = getPlugin(Backend);
+  const auto &ContextImpl = getSyclObjImpl(Context);
+
+  // Create PI properties from SYCL properties.
+  RT::PiQueueProperties Properties[] = {
+      PI_QUEUE_FLAGS,
+      queue_impl::createPiQueueProperties(
+          PropList, PropList.has_property<property::queue::in_order>()
+                        ? QueueOrder::Ordered
+                        : QueueOrder::OOO),
```

Comment:
the spec says:
> Also starting in version 4 the ```make_queue()``` function accepts a ```Properties``` member variable. This can contain any of the SYCL properties that are accepted by the SYCL queue constructor, except
the ```compute_index``` property which is built into the command queue or command list.

Does this really implement it this way? Should we error on `compute_index` and add support for `priority` property?

Reply:
Yes, it is implemented as described. "priority" is being handled. An error is issued for trying to use the compute_index property with make_queue.